docs/tutorial/from-zero-to-hero-write-your-first-kubernetes-charm/write-integration-tests-for-your-charm.md

```diff
@@ -130,33 +129,28 @@ async def test_database_integration(ops_test: OpsTest):
         apps=[APP_NAME], status='active', raise_on_blocked=False, timeout=120
     )
 ```
- 
 
 ```{important}
 
 But if you run the one and then the other (as separate `pytest ...` invocations, then two separate models will be created unless you pass `--model=some-existing-model` to inform pytest-operator to use a model you provide.
-
 ```
 
 In your Multipass Ubuntu VM, run the test again:
 
-
-```bash
+```text
 ubuntu@charm-dev:~/fastapi-demo$ tox -e integration
-
 ```
 
 The test may again take some time to run.
 
 ```{tip}
 
-**Pro tip:** To make things faster, use the `--model=<existing model name>` to inform `pytest-operator` to use the model it has created for the first test. Otherwise, charmers often have a way to cache their pack or deploy results; an example is https://github.com/canonical/spellbook . 
-
+**Pro tip:** To make things faster, use the `--model=<existing model name>` to inform `pytest-operator` to use the model it has created for the first test. Otherwise, charmers often have a way to cache their pack or deploy results; an example is [spellbook](https://github.com/canonical/spellbook).
```

Comment:
spellbook is a private repo.

given that the last update was 2 years ago, let's remove the second part of the pro tip. wdyt?

Reply:
Good plan, although it needs to go into @IronCore864's work I think, because he's reworking those chapters.